packages/rs-drive-abci/src/abci/handler/mod.rs

```diff
@@ -93,18 +93,39 @@ where
             )));
         }
 
+        let state_app_hash = state_guard
+            .last_block_app_hash()
+            .map(|app_hash| app_hash.to_vec())
+            .unwrap_or_default();
+
+        let latest_platform_version = PlatformVersion::latest();
+
         let response = proto::ResponseInfo {
             data: "".to_string(),
-            app_version: 1,
+            app_version: latest_platform_version.protocol_version as u64,
```

Comment:
No, this is wrong, the latest known platform version by your software isn't the current app version

Reply:
This is desired version so it must be the latest what you have